analysis/topEFT/work_queue_run.py

```diff
@@ -182,8 +182,8 @@ def LoadJsonToSampleName(jsonFile, prefix):
 
 executor_args = {
     'master_name': '{}-workqueue-coffea'.format(os.environ['USER']),
-
-    'xrootdtimeout': 180,
+    'filepath':'/tmp',
+    'xrootdtimeout': 300,
```

Comment:
@Andrew42, I think I'd prefer to keep the same default settings for the timeout and filepath committed for now. Is there a reason why you wanted to change these? Or were they maybe just accidentally committed with other updates?

Reply:
fixed with 80bbed74e99fe51bc931a15c9f7ddaec83a66c67